src/components/withPrepareCentralPaneScreen/index.tsx

```diff
@@ -0,0 +1,9 @@
+import type {ComponentType} from 'react';
+
+/**
+ * This HOC is dependent on platform. On native platforms screens that aren't already dipslayed in the navigation stack should be freezed to prevent from unnecessary rendering.
```

Comment:
```suggestion
 * This HOC is dependent on the platform. On native platforms, screens that aren't already displayed in the navigation stack should be frozen to prevent unnecessary rendering.
```